projects/baby_driver/scripts/gpio_set.py

```diff
@@ -0,0 +1,32 @@
+"""This module provides the Python implementation to set GPIO pins using the CAN protocol"""
+
+import can_util
+
+def gpio_set(port, pin, state):
+    """
+    Sets a GPIO pin
+    Args:
+        port: port of the GPIO pin to set
+        pin: pin number of the GPIO pin to set
+        state: 0 if setting low, 1 if setting high
+    Returns:
+        A boolean value. If true, the CAN message was sent successfully, otherwise
+        a ValueError or TimeoutError was raised.
+    """
+    if 0 <= port <= 6 and 0 <= pin <= 15 and 0 <= state <= 1:
+        can_pack_args = [(port, 1), (pin, 1), (state, 1)]
+    else:
+        return False
+
+    try:
+        can_util.send_message(babydriver_id=1, data=can_util.can_pack(can_pack_args))
+    except ValueError:
+        return False
+
+    try:
+        can_util.next_message(babydriver_id=1)
```

Comment:
You should be waiting for a status message, that's id 0. So use `BabydriverMessageId.STATUS` instead of 1 here. Also, `next_message` returns a Message object (see can_util.py), so please capture it in a variable and check its status (i.e. its `data[1]` attribute - see the confluence page linked in the ticket for a description of status messages). If the status returned by the status message is nonzero, that means something went wrong on the C side, so raise an exception specifying the nonzero status.